Gordon360/Models/ViewModels/RecIM/ParticipantActivityPatchViewModel.cs

```diff
@@ -3,9 +3,8 @@
 
 namespace Gordon360.Models.ViewModels.RecIM
 {
-    public class ParticipantPatchViewModel
+    public class ParticipantActivityPatchViewModel
```

Comment:
This view model seems to have two roles as far as relating data to other data goes. Could you explain further why a ParticipantActivity row should be used to relate a user to a team and to an activity? Shouldn't a team already have the relation to a specific activity?

If you wanted a separate participant-to-activity relationship table, that would make sense as you also have information that is specifically related to a user's privileges in an activity here, `ActivityPrivType`.